acq_stat_reports/acq_stat_reports.py

```diff
@@ -535,12 +582,12 @@ def main():
             "prev": f"{args.url}/{prev_range['year']}/{prev_range['subid']}/index.html",
         }
 
+        conf.data_dir = str(webout)
```

Comment:
Remember that this PR is for adding plots. I did change some other things, but that does not mean they will not change anymore.

In particular, I think there should not be two output directories. There should be only one, and it should be called `output_dir`. The split between "data" and "web" is not necessary, because all output from this package should be suitable for a web page, and that includes json files.

Anyway, I made some changes in this direction in #9 but there will be more changes, and I don't want to make these changes in this PR.